addons/base_vat/views/res_config_settings_views.xml

```diff
@@ -11,7 +11,8 @@
                          help="Verify VAT numbers using the European VIES service"
                          company_dependent="1"
                          title="If this checkbox is ticked, the default fiscal position that applies will depend upon the output of the verification by the European VIES Service."
-                         documentation="/applications/finance/accounting/taxation/taxes/vat_validation.html">
+                         documentation="/applications/finance/accounting/taxation/taxes/vat_validation.html"
+                         invisible="'EU' not in company_country_group_codes">
```

Comment:
not sure... You may be outside Europe but still want to make sure european partners get their VAT validated...

edit: or not... after a discussion, it appears it's arguable where to place the cursor but the main idea of company_country_group_codes is legit. So we'd better merge this code and finetune later, based on the feedback we'd receive.

Reply:
I thought VIES was for intrastat anyways, so outside of Europe would be less relevant anyways.